### PR TITLE
feature: Add Taplist.io cloud service support

### PR DESCRIPTION
**Pull request by TaplistIOTeam**
Tried to keep diffs low and follow the prevailing styles; welcome any feedback! We're certainly excited to support TiltBridge.

### CODE REVIEW

File: data/settings.htm [hunk 1]
```diff
@@ -531,6 +532,48 @@ <h5 class="card-title">Brewstatus Settings</h5>
...
+                                delete the Taplist.io URL.
+                            </p>
+                            <form action="/settings/taplistio/" method="POST" enctype="multipart/form-data">
+                                <div class="form-group row">
+                                    <label for="taplistioURL" class="col-sm-3 col-form-label" data-toggle="tooltip"
+                                        title="Target Taplist.io tiltbridge URL. Delete to disable pushing to Taplist.io">
+                                        Taplist.io URL</label>
+                                    <div class="col-sm-8">
+                                        <input type="url" class="form-control" name="taplistioURL" id="taplistioURL"
+                                            placeholder="https://api.taplist.io/api/tiltbridge/xxxx" :value="settings.taplistioURL"
```
[the remark this answers is not in the document]
Comment (TaplistIOTeam) [reply data/settings.htm]: > That said, is the placeholder URL for Tilt integrations properly https://api.taplist.io/api/**tiltbridge**/xxxx ?

Good catch - it is not! Fixed in bc35774.

**Comment by TaplistIOTeam** [comment]
Would you like me to rebase this against `devel`? I'm happy to do so.

**Comment by thorrak** [comment]
> Would you like me to rebase this against `devel`? I'm happy to do so.

Yes, please!

**Comment by TaplistIOTeam** [comment]
> Yes, please!

Done!

**Review by thorrak — APPROVED** [review]

**Pull request merged**